lib/ansible/modules/web_infrastructure/letsencrypt.py

```diff
@@ -92,6 +92,13 @@
     description: The destination file for the certificate.
     required: true
     aliases: ['cert']
+  chain_dest:
```

Comment:
Any reason you have the primary name as `chain_dest` and an alias of `chain`, then use the alias in the examples

Reply:
The primary name for the certificate filename is `dest`, with an alias of `cert`. That's why I used `chain_dest` as primary and `chain` as an alias.

The examples don't use `cert` either (but `dest`); since there's basically only one example (with three alternatives for the first step), either the primary or the alias should be used in all examples.

Reply:
Now that there's a second example using the aliases (see #35077), I've added this alias there as well. (I amended into the existing commit.)